_dprhtml/js/index.js

```diff
@@ -173,8 +177,12 @@ const loadHtmlFragmentAsync = (id, src, vm = null) =>
     });
   })
 
-const  historyPopstateHandler = e => {
+const historyPopstateHandler = async e => {
   console.warn('>>>> historyPopstateHandler', e);
+
+  if (e.currentTarget.location.search === '') {

```

Comment:
My thinking here is that no query string indicates that we should transition to the home page view. As far as I can tell, every other view is accompanied by a search string in the URL (e.g. `?loc=v.3.0.0.0.0.15.m`).

Note that this change does NOT handle navigation to other views, as my goal was to introduce minimal changes to resolve the specific reported issue.